src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/ConnectionPoint/ConnectionPointContainer.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using Microsoft.VisualStudio.OLE.Interop;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.ConnectionPoint
+{
+    /// <summary>
+    /// This implementation is a copy from CPS
+    /// </summary>
+    [ComVisible(true)]
+    public class ConnectionPointContainer : IConnectionPointContainer
```

Comment:
Even if we move this to a separate dll, we are still going to ship this new dll. Wouldn't the problem still be the same? except that the APIs are in a different dll(which can be excluded from Nuget but still shipped with VS)?